docker-compose.yml

```diff
@@ -104,11 +123,11 @@ services:
       context: ./
       dockerfile: ./docker/dotnet.dockerfile
     image: datadog-dotnet
-    command: bash -c "/project/docker/with-profiler-logs.bash wait-for-it elasticsearch5:9205 -- /project/docker/with-profiler.bash dotnet /project/samples/Samples.Elasticsearch.V5/bin/Release/netcoreapp2.1/publish/Samples.Elasticsearch.V5.dll"
+    command: bash -c "/project/docker/with-profiler-logs.bash wait-for-it elasticsearch5:9200 -- /project/docker/with-profiler.bash dotnet /project/samples/Samples.Elasticsearch.V5/bin/Release/netcoreapp2.1/publish/Samples.Elasticsearch.V5.dll"
     volumes:
     - ./:/project
     environment:
-    - ELASTICSEARCH5_HOST=elasticsearch5:9205
+    - ELASTICSEARCH5_HOST=elasticsearch5:9200
```

Comment:
Doesn't [this](https://github.com/DataDog/dd-trace-dotnet/pull/366/files#diff-4e5e90c6228fd48698d074241c2ba760R32) mean that port 9200 in the ES5 container is exported as port 9205 to the host? We can't run both ES5 and ES6 containers at the same time if they are both exported in the same host port, right?

Reply:
The elasticsearch5 container describes the following mapping: `localhost:9205`->`elasticsearch5:9200`. So this environment variable directly does the connection instead of reaching out to `localhost:9205`

Reply:
I based this off my Redis changes which are similar, and I was able to verify that behavior.